src/js/clickable-component.js

```diff
@@ -177,6 +177,18 @@ class ClickableComponent extends Component {
     this.off('keydown', this.handleKeyDown);
   }
 
+  /**
+   * Handles language change in ClickableComponent for the player in components
+   *
+   *
+   */
+  handleLanguagechange() {
+    if (!this.controlText_) {
+      return;
+    }
```

Comment:
`controlText()` does a null check for before proceeding, we can probably just call it unconditionally.